internal/services/pam/pam_test.go

```diff
@@ -0,0 +1,606 @@
+package pam_test
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"math/rand"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/authd"
+	"github.com/ubuntu/authd/cmd/authd/daemon"
+	"github.com/ubuntu/authd/internal/testutils"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var configFmt = `
+systemdirs:
+    socketpath: %s
+    brokercfgdir: %s
+`
+
+var (
+	fixturesPath = filepath.Join("testdata", "fixtures")
```

Comment:
We never used the directory fixtures in adsys. We used either the family/sub test name, or, if shared between multiple family tests, directly under the testdata/ (+ subdirectories if needed). please follow the same convention and let’s not add a new one :)